rosdep/base.yaml

```diff
@@ -1418,7 +1418,7 @@ geographiclib:
   openembedded: [geographiclib@meta-ros-common]
   rhel: [GeographicLib-devel]
   ubuntu:
-    '*': [libgeographiclib-dev]
+    '*': [libgeographic-dev]
```

Comment:
Looks like it's `libgeographiclib-dev` for Ubuntu Noble: https://packages.ubuntu.com/noble/libgeographiclib-dev. So I'm not sure this change is needed.

Reply:
I agree that the rosdep key is correct as is.

focal, jammy [libgeographic-dev](https://packages.ubuntu.com/jammy/libgeographic-dev)
Noble and beyond [https://packages.ubuntu.com/noble/libgeographiclib-dev](https://packages.ubuntu.com/noble/libgeographiclib-dev)